src/odemis/driver/xt_client.py

```diff
@@ -2170,7 +2196,7 @@ def __init__(self, name: str, role: str, parent: model.HwComponent, rng: dict =
             "rz": model.Axis(unit=stage_info["unit"]["r"], range=rng["rz"]),
         }
 
-        model.Actuator.__init__(self, name, role, parent=parent, axes=axes_def,
+        model.Actuator.__init__(self, name, role, parent=self.parent, axes=axes_def,
```

Comment:
This function takes care of setting `self.parent` as `parent`. So no need to do it early explicitly.